xcodeproj/internal/xcode_schemes.bzl

```diff
@@ -147,6 +160,13 @@ def _unfocus_schemes(schemes, unfocused_targets):
 
     return focused_schemes
 
+def _pre_post_action(name, target, script_contents):
+    return struct(
+        name = name,
+        target = target,
```

Comment:
This can be `None`, right? Pre/post actions don't require (though usually use) an `environmentBuildable`.

Reply:
Yes, it can be None. Updated to keep the naming consistent with test action - "expand_variables_based_on"